optaplanner-examples/src/main/resources/org/optaplanner/examples/meetingscheduling/solver/meetingSchedulingScoreRules.drl

```diff
@@ -112,5 +121,47 @@ rule "Do all meetings as soon as possible"
     when
         MeetingAssignment(startingTimeGrain != null, $lastTimeGrainIndex : lastTimeGrainIndex)
     then
-        scoreHolder.addSoftConstraintMatch(kcontext, - $lastTimeGrainIndex);
+        scoreHolder.addSoftConstraintMatch(kcontext, - $lastTimeGrainIndex % 100 * MeetingSchedulingConstraints.DO_ALL_MEETINGS_AS_SOON_AS_POSSIBLE.getConstraintWeight());
+end
+
+//TODO: Add minimum break in time grains to class MeetingSchedule. Minimum break here is 1 timeGrain
+//TODO: replace hard-coded values with constraint weights gotten from Configuration sheet
+rule "Minimum TimeGrains break between two consecutive meetings"
+    when
+        MeetingAssignment($leftMeeting : meeting, $leftRoom : room, startingTimeGrain != null, $leftEndTimeGrainIndex : getLastTimeGrainIndex())
+        MeetingAssignment(startingTimeGrain != null, meeting != $leftMeeting, $rightStartingTimeGrainIndex : startingTimeGrain.getGrainIndex(),
+            $leftEndTimeGrainIndex < $rightStartingTimeGrainIndex, $rightStartingTimeGrainIndex - $leftEndTimeGrainIndex - 1 < 1)
```

Comment:
How about something like

```
when
  MeetingAssignment(startingTimeGrain != null, $room : room,  $leftEnd : getLastTimeGrainIndex())
  MeetingAssignment(room == $room, startingTimeGrain != null, $leftEnd == getFirstTimeGrainIndex() - 1)
```

Reply:
This will work and is 3x faster, although we won't be able to penalize for a break of more than one timeGrain, but I think it makes sense for a break to be only one timeGrain.